sdk/documentintelligence/Azure.AI.DocumentIntelligence/samples/Sample_ExtractLayout.md

```diff
@@ -1,8 +1,8 @@
-# Analyze with the prebuilt general document model
+# Extract the layout of a document
 
-This sample demonstrates how to analyze key-value pairs, tables, and selection marks from documents using the prebuilt general document model.
+This sample demonstrates how to extract text, paragraphs, styles, table structures, and selection marks, along with their bounding region coordinates from documents.
 
-To get started you'll need a Cognitive Services resource or a Document Intelligence resource. See [README][README] for prerequisites and instructions.
+To get started you'll need a Cognitive Services resource or a Form Recognizer resource. See [README][README] for prerequisites and instructions.
```

Comment:
Is the resource still called "Form Recognizer"? If so, we should edit it in the other samples too.

Reply:
It's not. Good catch!